### ISSUE TITLE
demo notebook fails: pyodide.js not found

### ISSUE THREAD

**Issue by jeremydouglass** [iodide-project/pyodide#803]
The README links to a demo notebooks at:

https://alpha.iodide.io/notebooks/300/

Running the demo notebook fails at

```
%% py
# python
import sys
```

with error:

> Python plugin failed to load: https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js not found

**Comment by rth** [comment]
Thanks for the report. I can't reproduce, the demo iodide notebook works fine for me. 

Are you able to load the `pyodide.js` file directly? Maybe it's blocked by some browser plugin?

**Comment by jeremydouglass** [comment]
Odd. I can load the file directly at https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js

However, in Chrome 86, this is how the error is displayed in the notebook interface:

<img width="1143" alt="Screen Shot 2020-11-19 at 1 23 28 PM" src="https://user-images.githubusercontent.com/798570/99725978-c41bfa80-2a6a-11eb-83f9-2d43a6674fa1.png">

I just checked the javascript console, and it looks like a CORS problem.

> (index):1 Access to XMLHttpRequest at 'https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js' from origin 'https://alpha.iodide.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
> pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js:1 Failed to load resource: net::ERR_FAILED

Is pyodide only supposed to work with Firefox?

**Comment by rth** [comment]
> Is pyodide only supposed to work with Firefox?

No it should work. I also can't reproduce with Chrome 84 on  alpha.iodide.io/notebooks/300

We had some similar issues in the past https://github.com/iodide-project/pyodide/issues/669#issuecomment-631783225 but it should no longer be an issue.

Can you check that in the requests that are sent there is indeed an `Origin` header? In which case there should be a `Access-Control-Allow-Origin` response header.

**Comment by jeremydouglass** [comment]
I checked again this afternoon and the notebook ran fine this time -- no CORS error. This is in the same running browser session, with computer and tab set not restarted from three days ago when I filed this issue, same local network, no manual change in Chrome extensions, Chrome not updated....

I'm going to guess (?) that this was some temporary server-side issue. My second guess would be that it was a behavior related to an installed extension on my local machine. I'll re-open with more details if it comes up again.
